### DISCUSSION TITLE
Environment variables are not getting updated.

### DISCUSSION THREAD

**Discussion by cyb0rg14** [vercel/next.js#59547] [Help]
### Summary

Even if I'm updating my environment variables on my hosted vercel site, it still not using latest values and rather stuck on my past values of env keys. Is it some problem of vercel or I have to do something to resolve this issue ?? Help me if you know anything related to this issue.

**Comment by pybern** [comment]
Do you have an example of what environmental variables you want to pass? A typical root cause is when `NEXT_PUBLIC` is not prefixed and therefore will not be exposed on client side. You can read more [here](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#bundling-environment-variables-for-the-browser).
